lib/livebook/config.ex

```diff
@@ -278,7 +278,7 @@ defmodule Livebook.Config do
     identity_logout? =
       Code.ensure_loaded?(module) and function_exported?(module, :logout, 2)
 
-    authentication().mode != :disabled or identity_logout?
+    identity_logout?
```

Comment:
You no longer need the variable, you can return `Code.ensure_loaded?(module) and function_exported?(module, :logout, 2)` directly instead.

Reply:
Yes, sometimes I prefer assigning things to a variable, just to give semantics to what the code is doing. Extracting to a function is another way, but maybe it is too much here?

I'm ok with not using the variable, though, but I would still prefer to make it easier for future readers what is being done here.

Suggestions?

Reply:
Ah, if this is a style choice, I am fine either way.

Reply:
For that specific case, I think it's ok not having an intermediate variable. Removed it. 👍